src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -11,34 +14,54 @@ public class Exercise {
 
     public void stepOne() {
         // TODO: 1. Write a for loop that adds the numbers 0 to 3 to the numsZeroToThree array
+        this.numsZeroToThree = new int[4];
+        for(int i = 0; i < 4; i++) {
+            this.numsZeroToThree[i] = i;
 
+        }
 
     }
 
     public void stepTwo() {
         // TODO: 2. Write a for loop that adds the numbers 5 to 10 to the numsFiveToTen array
 
+        this.numsFiveToTen = new int[6];
+        for(int i = 0; i < 6; i++) {
```

Comment:
You could also initialise i at the desired index. `for(int i = x;...`. Where x is the desired index

Reply:
I did that first, but than remembered that i had to than do:

numsFiveToTen[i-5] = i;

Unless there is better way to assign value to index on string[] array . If it was an arraylist i would just add to it from 5 to 10.

Reply:
Yeah, that's right, you would need to do:

```
for (int i = 5; i < 11; i++) {
    this.numsFiveToTen[i - 5] = i;
}
```

I saw after you initialised `i` at a different number so a slightly redundant comment if you already know about this ^^